superset/db_engine_specs/ocient.py

```diff
@@ -28,6 +31,36 @@
 pyocient.logger.setLevel(superset_log_level)
 
 
+# Regular expressions to catch custom errors
+
+CONNECTION_INVALID_USERNAME_REGEX = re.compile(
+    "The referenced user does not exist \(User \'(?P<username>.*?)\' not found\)"
+)
+CONNECTION_INVALID_PASSWORD_REGEX = re.compile(
+    'The userid/password combination was not valid \(Incorrect password for user\)'
+)
+CONNECTION_INVALID_HOSTNAME_REGEX = re.compile(
+    r"Unable to connect to (?P<host>.*?):(?P<port>.*?):"       
+)
+CONNECTION_UNKNOWN_DATABASE_REGEX = re.compile(
+    "No database named '(?P<database>.*?)' exists"
+)
+CONNECTION_INVALID_PORT_ERROR = re.compile(
+    "Port out of range 0-65535"
+)
+INVALID_CONNECTION_STRING_REGEX = re.compile(
+    "An invalid connection string attribute was specified \(failed to decrypt cipher text\)"
+)
+SYNTAX_ERROR_REGEX = re.compile(
+    r"There is a syntax error in your statement \(extraneous input '(?P<ext_input>.*?)' expecting {.*}"
```

Comment:
did you get a chance to confirm this latest change still matches?

Reply:
oh nvm, I have my UT!

Reply:
will test and report back

Reply:
yea, going to need to handle the other case...

Reply:
Fixed